Pearson-Advance/openedx-themes
# Add default pearson-theme.

## Description

This PR adds a default theme for ecommerce.

## Testing:

- Enable comprehensive theming.
- Run the LMS: `make dev.up.lms`.
- Set `pearson-theme` on a ecommerce site.
- Compile the dafault theme, `make static` in ecommerce shell.
- Verify that the default theme is applied correctly on Olive.

(For more information enter here: [How to enable themes)](https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/changing_appearance/theming/enable_themes.html#enable-themes)

## Screenshots

Basket:

![image](https://github.com/Pearson-Advance/openedx-themes/assets/30726391/a1225532-b599-4ad6-ad9e-b2aa3291c20c)

Receipt:

![image](https://github.com/Pearson-Advance/openedx-themes/assets/30726391/6770dc7e-890c-4b7c-8ffc-02b4601db787)
